tests/lib/external/snapd-testing-tools/COPYING

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
I see this commit imports some shared code from somewhere else. What is the procedure to keep track of updates?

Reply:
As we talked about the code in that folder is a subtree